nitrogen/src/submit_group.rs

```diff
@@ -159,8 +160,17 @@ impl SubmitGroup {
     }
 
     /// Setup a new swapchain for a given `Display`, recreating it if an old one already exists.
-    pub unsafe fn display_setup_swapchain(&mut self, ctx: &mut Context, display: DisplayHandle) {
-        ctx.displays[display].setup_swapchain(&ctx.device_ctx, &mut self.res_destroys);
+    pub unsafe fn display_setup_swapchain(
+        &mut self,
+        ctx: &mut Context,
+        display: DisplayHandle,
+        swapchain_setup_config: &SwapchainSetupConfig,
```

Comment:
Now that the type is `Copy` I think we can just pass by value here and don't need the reference